frontend/src/pages/Register.vue

```diff
@@ -15,7 +15,7 @@
     </div>
 
     <!-- Page content -->
-    <b-container class="mt--8 p-1">
+    <b-container v-if="!showPageMessage" class="mt--8 p-1">
```

Comment:
v-if="!showPageMessage" / v-else

I would change, so that there is no need for a negative query.

Reply:
same same 👆🏼

Reply:
No negation

Reply:
Done
Thanks for your hint @ogerly